dae/dae/testing/import_helpers.py

```diff
@@ -17,7 +18,23 @@ class StudyLayout:
     cnv: list[pathlib.Path]
 
 
-def data_import(root_path: pathlib.Path, study: StudyLayout, gpf_instance):
+def update_study_config(gpf_instance, study_id: str, study_config_update: str):
+    """Update study configuration."""
+    config_path = pathlib.Path(gpf_instance.dae_dir) / \
+        "studies" / \
+        study_id / \
+        f"{study_id}.yaml"
+    with open(config_path, "r", encoding="utf") as infile:
+        config = yaml.safe_load(infile.read())
+    config_update = yaml.safe_load(study_config_update)
+    config.update(config_update)
+    with open(config_path, "wt", encoding="utf8") as outfile:
```

Comment:
Shouldn't we use StudyConfigBuilder here?

Reply:
I agree it would be better to use `StudyConfigBuilder.` I created a new issue to track this work: #303